src/components/navbar/Navbar.jsx

```diff
@@ -25,12 +25,14 @@ const Navbar = () => {
           <p className="hover:text-darkGrayishBlue">How does it work?</p>
           <p className="hover:text-darkGrayishBlue">About us</p>
           {/* <p >|</p> */}
-          <p
-            className="hover:text-darkGrayishBlue pl-3 py-2"
-            style={{ borderLeftWidth: "1px" }}
-          >
-            Sign in
-          </p>
+          <Link to="/signin">
+            <p
+              className="hover:text-darkGrayishBlue pl-3 py-2"
+              style={{ borderLeftWidth: "1px" }}
+            >
+              Sign in
+            </p>
+          </Link>
```

Comment:
can you add a LInk to the sign in in the hamburger menu as well? below <p className=“hover:text-darkGrayishBlue”>Sign In</p>